src/content-scripts/components/GenerateAIDescription/DescriptionGeneratorButton.ts

```diff
@@ -1,12 +1,12 @@
 import { createHtmlElement } from "../../../utils/createHtmlElement";
 import openSaucedLogoIcon from "../../../assets/opensauced-icon.svg";
 import { getPullRequestAPIURL } from "../../../utils/urlMatchers";
-import { getDescriptionContext, isContextWithinBounds } from "../../../utils/fetchGithubAPIData";
+import { getDescriptionContext, isOutOfContextBounds } from "../../../utils/fetchGithubAPIData";
```

Comment:
I liked the previous name better, but this looks good overall

Reply:
Same

Reply:
Previously the function was returning `Boolean(isWithinTokemLimit(...))`.

The `isWithinTokenLimit(...)` returns `false | number`, the number being the length of the tokens. If the length of the tokens was zero, the `isContextWithinBounds()` would return `false`. Which was incorrect. I've updated the return value to ````isWithinTokenLimit()` === false`. Hence the rename.

I've added a separate check to see if the context is empty, https://github.com/open-sauced/ai/blob/a5b7cddeec0d26e226aefcffd14e993dbb3b0b9b/src/content-scripts/components/GenerateAIDescription/DescriptionGeneratorButton.ts#L45.